inspirehep/modules/literaturesuggest/tasks.py

```diff
@@ -218,8 +218,8 @@ def _is_arxiv_url(url):
 
 def new_ticket_context(user, obj):
     """Context for literature new tickets."""
-    title = get_title(obj.data)
-    subject = "Your suggestion to INSPIRE: {0}".format(title)
+    title = get_title(obj.data).decode('utf-8')
```

Comment:
That was erroneous and it was required by this `u'title':` that I had typed in test `data`.